models/article-model.js

```diff
@@ -43,4 +43,28 @@ function fetchArticleComments(id) {
     });
 }
 
-module.exports = { fetchArticleById, fetchAllArticles, fetchArticleComments };
+function patchAlterVotes(votes, id) {
+  if (typeof votes != "number") {
+    return Promise.reject({ status: 400, msg: "Invalid request" });
+  }
```

Comment:
No need for the if statement as SQL will shout if the article id is not a number